src/pytsql/grammar/tsqlParser.py

```diff
@@ -24890,6 +24890,13 @@ def getRuleIndex(self):
             return tsqlParser.RULE_data_type
 
 
+    @staticmethod
+    def is_top_level_statement(node: ParserRuleContext):
+        """Check wether node is a top level SQL statement."""
+        cur = node.parentCtx
+        while isinstance(cur, tsqlParser.Sql_clauseContext) or isinstance(cur, tsqlParser.Sql_clausesContext):
+            cur = cur.parentCtx
+        return isinstance(cur, tsqlParser.BatchContext)
```

Comment:
I think this file is autogenerated from the ANTLR grammar and we shouldn't modify it manually